nuitka/utils/WindowsResources.py

```diff
@@ -155,6 +155,7 @@ def callback2(hModule2, lpType2, lpName2, wLang, _lParam):
 
 
 def _openFileWindowsResources(filename):
+    absFilename = os.path.abspath(filename)
```

Comment:
Do you recall, why the `abspath` calls were necessary? I normally try to avoid them as they can only cause length issues and indetermism.

Reply:
Without the abspath, it threw a file not found type of error.

Reply:
I have avoided absolute paths typically, because it's easy to run into length limits for no good reason. Make sure it's not a locking issue with Windows Defender, that is actually more often the case, but I trust you there.